src/components/application_manager/rpc_plugins/app_service_rpc_plugin/src/app_service_app_extension.cc

```diff
@@ -38,7 +38,8 @@ SDL_CREATE_LOG_VARIABLE("AppServiceRpcPlugin")
 
 namespace app_service_rpc_plugin {
 
-const AppExtensionUID AppServiceAppExtensionUID = 455;
+const app_mngr::AppExtensionUID
+    AppServiceAppExtension::AppServiceAppExtensionUID = 455;
```

Comment:
@OlhaVorobiova I think you can move this initialization to a header file as you did for `rc_app_extension`